doc/troubleshooting.md

```diff
@@ -58,3 +60,27 @@ mount: wrong fs type, bad option, bad superblock on 10.100.200.247:/export/pvc-6
        In some cases useful info is found in syslog - try
        dmesg | tail or so.
 ```
+
+## Recover Statefulset when node fails
+
+When a node failure happens, unlike Deployment Pod, the Statefulset Pod on that failed node will not be rescheduled to other available nodes automatically.
+This is because Kubernetes guarantees at most once execution of a Statefulset.
```

Comment:
Need to add a reference to the Kubernetes documentation where this is mentioned.